src/pages/registration/resource_type_tab/SelectRegistrationTypeField.tsx

```diff
@@ -220,79 +228,166 @@ export const SelectRegistrationTypeField = () => {
             </IconButton>
           )}
         </Box>
-        <Box sx={{ display: 'flex', gap: '0.5rem', alignItems: 'center', justifyContent: 'end', mb: '1rem' }}>
-          <FilterVintageIcon
-            color="primary"
-            titleAccess={t('registration.resource_type.nvi.can_give_publication_points')}
-            fontSize="small"
+        <Paper
+          elevation={5}
+          sx={{
+            padding: '0.5rem',
+            bgcolor: 'secondary.main',
+            display: 'grid',
+            gap: '0.5rem',
+            gridTemplateAreas: {
+              xs: "'search-bar' 'publication-points-text'",
+              md: "'search-bar publication-points-text'",
+            },
+            gridTemplateColumns: { xs: '1fr', md: '1fr 1fr' },
+            mb: '1rem',
+          }}>
```

Comment:
Syns det ble litt mye fargegreier her nå 🤔 I skissene gir det litt mer mening siden det er en del av alternating color, men når vi ikke har det ende syns jeg det ble litt for dominant på en måte.
![bilde](https://user-images.githubusercontent.com/6313468/220297855-d13adeae-b518-43c8-9c93-149c76591e71.png)

Lurer på om dette kan løses enda enklere med en flex også? Feks noe slik:

```suggestion
        <Box
          sx={{
            display: 'flex',
            justifyContent: 'space-between',
            flexWrap: 'wrap',
            gap: '0.5rem',
            my: '1rem',
          }}>
```